packages/graphql/src/translate/queryAST/factory/Operations/ReadFactory.ts

```diff
@@ -189,17 +180,10 @@ export class ReadFactory {
         resolveTree: ResolveTree,
         context: Neo4jGraphQLTranslationContext
     ) {
-        // SOFT_DEPRECATION: OPTIONS-ARGUMENT
-        const optionsArg: Record<string, any> = (resolveTree.args.options ?? {}) as Record<string, any>;
-        const sortArg = resolveTree.args.sort ?? optionsArg.sort;
-        const limitArg = resolveTree.args.limit ?? optionsArg.limit;
-        const offsetArg = resolveTree.args.offset ?? optionsArg.offset;
-        raiseOnMixedPagination({
-            optionsArg,
-            sort: resolveTree.args.sort,
-            limit: resolveTree.args.limit,
-            offset: resolveTree.args.offset,
-        });
+        const sortArg = resolveTree.args.sort as Record<string, any>[];
+        const limitArg = resolveTree.args.limit as number;
+        const offsetArg = resolveTree.args.offset as number;
+
```

Comment:
the block below is not longer necessary

Reply:
I think it may be! Despite being called `getOptions`, I still think it deals with applying sorting and pagination. Otherwise this function is literally just assigning values to three variables and then returning nothing! 😆